src/stream/src/executor/hash_agg.rs

```diff
@@ -670,580 +670,3 @@ impl<K: HashKey, S: StateStore> HashAggExecutor<K, S> {
         }
     }
 }
-
-#[cfg(test)]
-pub mod tests {
-    use assert_matches::assert_matches;
-    use futures::StreamExt;
-    use risingwave_common::array::stream_chunk::StreamChunkTestExt;
-    use risingwave_common::array::StreamChunk;
-    use risingwave_common::catalog::{Field, Schema};
-    use risingwave_common::types::DataType;
-    use risingwave_expr::agg::{AggArgs, AggCall, AggKind};
-    use risingwave_storage::memory::MemoryStateStore;
-    use risingwave_storage::StateStore;
-
-    use crate::executor::test_utils::agg_executor::new_boxed_hash_agg_executor;
-    use crate::executor::test_utils::*;
-    use crate::executor::{Message, PkIndices, Watermark};
-
-    // --- Test HashAgg with in-memory StateStore ---
-
-    #[tokio::test]
-    async fn test_local_hash_aggregation_count_in_memory() {
-        test_local_hash_aggregation_count(MemoryStateStore::new()).await
-    }
-
-    #[tokio::test]
-    async fn test_global_hash_aggregation_count_in_memory() {
-        test_global_hash_aggregation_count(MemoryStateStore::new()).await
-    }
-
-    #[tokio::test]
-    async fn test_local_hash_aggregation_min_in_memory() {
-        test_local_hash_aggregation_min(MemoryStateStore::new()).await
-    }
-
-    #[tokio::test]
-    async fn test_local_hash_aggregation_min_append_only_in_memory() {
-        test_local_hash_aggregation_min_append_only(MemoryStateStore::new()).await
-    }
-
-    async fn test_local_hash_aggregation_count<S: StateStore>(store: S) {
```

Comment:
This test is deleted. Is that intended?

Reply:
Yes, this test looks the same as another so called _global_ one.